ci/build_docs.sh

```diff
@@ -25,7 +25,9 @@ rapids-mamba-retry install \
   --channel "${PYTHON_CHANNEL}" \
   kvikio libkvikio
 
-export RAPIDS_VERSION_NUMBER="24.04"
+export RAPIDS_VERSION="$(rapids-version)"
+export RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
+export RAPIDS_VERSION_NUMBER="$RAPIDS_VERSION_MAJOR_MINOR"
```

Comment:
is this one still used anywhere?

Reply:
Yes, it's used by `rapids-upload-docs`.

Reply:
Have been confused by the same thing before

Maybe [this reference]( https://github.com/rapidsai/gha-tools/blob/a75d2b5dd1b01129490b478e38f18eb6e35170ac/tools/rapids-upload-docs#L15-L17 ) helps

Honestly think we should consider moving away from requiring environment variables in scripts and switch to adding these as script parameters. It is much easier to understand how the scripts are used then. That's a separate discussion though